src/Workspaces/CoreTest/LinkedFileDiffMerging/LinkedFileDiffMergingTests.TextMerging.cs

```diff
@@ -44,15 +44,15 @@ public void TestIsolatedChangesInBothFiles()
                 LanguageNames.CSharp);
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/44423")]
+        [Fact]
         [Trait(Traits.Feature, Traits.Features.LinkedFileDiffMerging)]
         [WorkItem(44423, "https://github.com/dotnet/roslyn/issues/44423")]
         public void TestIdenticalEditAfterIsolatedChanges()
         {
             TestLinkedFileSet(
-                "a b c d e",
-                new List<string> { "a zzz c xx e", "a b c xx e" },
-                @"a zzz c xx e",
+                "a; b; c; d; e;",
+                new List<string> { "a; zzz; c; xx; e;", "a; b; c; xx; e;" },
+                @"a; zzz; c; xx; e;",
                 LanguageNames.CSharp);
```

Comment:
i'm a little curious about what broke here and why we need the input to the test to change.

Reply:
It's based on the C# syntax tree, and the previous input didn't parse consistently with the new feature.